packages/jellyfish-api-core/__tests__/category/masternode/isAppliedCustomTransaction.test.ts

```diff
@@ -0,0 +1,93 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+
+describe('Masternode', () => {
+  const container = new MasterNodeRegTestContainer()
```

Comment:
try to use `testing.container`, we plan to put the container creation inside `testing` class.
you can find sample code in loan tests.
@chee-chyuan FYI